dspace-api/src/test/java/org/dspace/core/EmailTest.java

```diff
@@ -65,4 +69,189 @@ public void testNotNullParameter()
+
+        // Check that original recipient is included in the body
+        String messageBody = message.getContent().toString();
+        assertThat("Original recipient should be included in email body",
+                messageBody, containsString("original@example.com"));
+        assertThat("Email body should contain REAL RECIPIENT marker",
+                messageBody, containsString("===REAL RECIPIENT==="));
+    }
+
+    @Test
+    public void testMultipleCatchAllRecipientsWhenMailServerDisabled()
+            throws MessagingException, IOException {
+        config.setProperty("mail.server.disabled", "true");
+        config.setProperty("mail.server.catchAll.enabled", "true");
+        config.setProperty("mail.server.catchAll.recipient", "fixed1@example.com,fixed2@example.com");
+
+        Email email = new Email();
+        email.setContent("multiple catchAll recipients test",
+                "This is a test email that should go to multiple catchAll recipients.");
+        email.addRecipient("original@example.com");
+        email.build();
+
+        MimeMessage message = email.message;
+        InternetAddress[] recipients = (InternetAddress[]) message.getRecipients(MimeMessage.RecipientType.TO);
+
+        assertThat("Email should be sent to both catchAll recipients", recipients.length, is(2));
+        assertThat("First catchAll recipient should receive the email", recipients[0].getAddress(),
+                   is("fixed1@example.com"));
+        assertThat("Second catchAll recipient should receive the email", recipients[1].getAddress(),
+                   is("fixed2@example.com"));
+    }
+
+    @Test
+    public void testDisabledServerWithoutCatchAllRecipient()
+            throws MessagingException, IOException {
+        config.setProperty("mail.server.disabled", "false");
+        config.setProperty("mail.server.catchAll.enabled", "true");
+        config.setProperty("mail.server.catchAll.recipient", "");
+
+        Email email = new Email();
+        email.setContent("disabled server test",
+                "This is a test email that should not be sent.");
+        email.addRecipient("original@example.com");
+        email.build();
+
+        MimeMessage message = email.message;
+        InternetAddress[] recipients = (InternetAddress[]) message.getRecipients(MimeMessage.RecipientType.TO);
+
+        assertThat("Message should be created", message, is(not((MimeMessage) null)));
+        assertThat("Email should be sent to original recipient when no catchAll recipient configured",
+                recipients.length, is(1));
+        assertThat("Original recipient should receive the email", recipients[0].getAddress(),
+                   is("original@example.com"));
+
+        // Check that original recipient is NOT included in the body (normal behavior)
+        String messageBody = message.getContent().toString();
+        assertThat("Original recipient should NOT be included in email body for normal disabled behavior",
```

Comment:
Confusing assertion label:  the test is for the REAL RECIPIENT header, not for the original recipient.  The comment above is also not quite correct.